NevisExampleApp/Domain/Validators/AccountValidatorImpl.swift

```diff
@@ -19,7 +19,7 @@ extension AccountValidatorImpl: AccountValidator {
 		}
 
 		var accounts = [any Account]()
-		supportedAuthenticators.forEach { authenticator in
+		for authenticator in supportedAuthenticators {
```

Comment:
I think the SwiftFormat rule is missing here not to rewrite the forEach

Reply:
Fixed.